JS-Algorithms/challenges.js

```diff
@@ -118,9 +150,24 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
-};
+  let n1 = 0, n2 = 1, nextTerm;
+  let result = [0, 1]
+
```

Comment:
is this really necessary? 🤔, I think you can just store the last fibonacci number